src/library/scala/collection/immutable/Vector.scala

```diff
@@ -952,7 +952,6 @@ private[immutable] trait VectorPointer[T] {
 
     private[immutable] final def copyOf(a: Array[AnyRef]) = {
       //println("copy")
```

Comment:
Sure, I can do that, but let me explain why I didn't do it in the first place: given the zillions of other commented out `println`s in this file, I didn't want to intrude too much. Supposedly those `println`s were useful for tracing purposes in the event of a debugging war against this complex data structure. The `a eq null` case immediately crashes on the next line, so it's no big deal to remove it. But supposedly tracing `copy` was instrumental to someone's efforts to debugging this class at some point. If I remove it but don't remove the other hundreds of `println`s, maybe I'll actually be *harming* a future someone trying to debug it again, because they won't realize that `copy` is worth tracing.